prow/cluster/cherrypicker_deployment.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright 2018 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: cherrypicker
+  labels:
+    app: cherrypicker
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: cherrypicker
+    spec:
+      terminationGracePeriodSeconds: 180
+      containers:
+      - name: cherrypicker
+        image: gcr.io/k8s-prow/cherrypicker:latest
+        imagePullPolicy: Always
+        args:
+        - --dry-run=false
+        ports:
+          - name: http
+            containerPort: 8888
+        volumeMounts:
+        - name: hmac
+          mountPath: /etc/webhook
+          readOnly: true
+        - name: oauth
+          mountPath: /etc/github
+          readOnly: true
+      volumes:
+      - name: hmac
+        secret:
+          secretName: hmac-token
+      - name: oauth
+        secret:
+          secretName: oauth-token
```

Comment:
I would suggest using a bot different from k8s-ci-robot. All it needs is org membership in order for its PRs to get automatically tested but other than that it shouldn't need any write permissions in repos.

Reply:
fejta-bot perhaps? :upside_down_face: 
thoughts @fejta?

Reply:
If we do decide to use that bot do we just replace "oauth-token" with "fejta-bot-token" ?

Reply:
Or do we need a new bot / token

Reply:
fejta-bot sgtm assuming it needs no permissions to anything aside from a github account

Reply:
do we need to set up a new token for it, or is there an existing one?

I can't find any mention of fejta-bot in this repo, aside from a few comments.

Reply:
/cc @fejta

Reply:
Search prow/config.yaml for prior art:

https://github.com/kubernetes/test-infra/blob/5b2902d5a7982c579b7a9fd4a0a82a067ba3524d/prow/config.yaml#L13705-L13716